.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Device Request
+about: This form helps in requesting a new device to be supported.
+title: "[DEVICE SUPPORT REQUEST] - Devicename"
+labels: Device Request
+assignees: ''
+
+---
+
+Before requesting a device, please make sure to search the open and closed issues for any requests in the past. Sometimes devices have been requested before but are not implemented yet due to various reasons. If there are no hits on your Product Number, please proceed.  
+
+**Device Name:**
+
+**Device Type:**
+- [ ] Light
+- [ ] Lock
+- [ ] Remote
+- [ ] Sensor
+- [ ] Siren
+- [ ] Thermostat
+- [ ] Other: 
+
+**Device Manufaturer:**
+
+**Device Product Number:**
+
+**If you do not know what is needed, please read [this](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Request-Device-Support)**
+
+**Screenshot of Clusters of the Node:**
+
+**Screenshot of the node info panel:**
+
+**Screenshot of the basic clusters attributes as shown in the Clusters info panel with the data returned by read:** 
+ *Please note that at least "Manufacturer Name" and "Model Identifier" must be populated with data (therefore, must not be empty), otherwise that information will not be usable. For battery powered devices, after pressing read it is required to wake-up the device by pressing a button or any other means of interaction.*
+
+**Screenshots of the other clusters (if available:**
+*Further clusters (if available) including Simple Metering, Electrical Measurement, Power Configuration, Thermostat. Again, please ensure data has been read prior to taking any screenshots.*
```

Comment:
Maybe better to list what to exclude? Identify, groups, alarms...

Also, same hint for reading the attributes before taking screenshots. Better also include a hint how to wake light/deep sleepers before reading

Reply:
I just coppied this from your guide in the discord channel, can you rewrite it ?

Reply:
Slightly updated the wiki, you may just copy that over.

Reply:
Done!